app/models/survey-model.js

```diff
@@ -35,7 +95,7 @@ if ( process.env.NODE_ENV === 'test' ) {
  * @name get
  * @function
  * @param { string } id - Survey ID
- * @return {Promise<Error|SurveyObject>} Promise that resolves with a survey object
+ * @return {Promise<SurveyObject>} Promise that resolves with a survey object
```

Comment:
While I wish there were some way to express a Promise’s rejection types, this unfortunately wasn’t accurate and may be confusing at call sites.